knex/seeds/01_users.js

```diff
@@ -0,0 +1,21 @@
+exports.seed = (knex) => {
+  const foreignKeys = 'ALTER TABLE "users_projects" DROP CONSTRAINT IF EXISTS "users_projects_fk0"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk0"; ALTER TABLE "users_transcripts" DROP CONSTRAINT IF EXISTS "users_transcripts_fk0";';
```

Comment:
like mentioned before - backticks so this could go over a few lines would be good. Shame the `dropForeign` has gone?